src/app/components/loaders/BasicSpinner.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import "./BasicSpinner.scss";
+
+const BasicSpinner: React.FC = () => {
```

Comment:
just a hint, everything is okay:
You could have used MUI spinner component to save time instead of developing ur custom spinner, but it seems to fit better our page style